crates/plugin/src/config.rs

```diff
@@ -26,6 +26,18 @@ pub struct Config {
     /// Unused but required by the validator to load the plugin
     #[allow(dead_code)]
     libpath: String,
+
+    /// Unused here but is in validator
```

Comment:
so is it even needed here?

Reply:
Parsing error due to `deny_unknown_fields`